jit/v2/JIT_v2.0.0.yaml

```diff
@@ -191,7 +253,13 @@ paths:
       operationId: omit-port-call
       summary: Omits a Port Call
       description: |
-        Provider `OMITTING` a **Port Call**
+        Provider `OMITTING` a **Port Call**. This is called whenever the provider has to omit a **Port Call**.
+
+        When a consumer reieves an OMIT - it is the responsibility to pass on this information to any secondary receivers the consumer has called in the past using the `isFYI=true` property while creating **Port Calls** also using `isFYI=true`.
```

Comment:
When a consumer receives an `OMIT`, it is their responsibility to propagate this information to any secondary recipients they previously informed using the `isFYI=true` property while creating **Port Calls** or sending related updates.